client/lib/plugins/actions.js

```diff
@@ -211,6 +211,10 @@ PluginsActions = {
 			return queueSitePluginActionAsPromise( wpcom.pluginsInstall.bind( wpcom ), site.ID, plugin.slug );
 		};
 
+		update = function( pluginData ) {
+			return queueSitePluginActionAsPromise( wpcom.pluginsUpdate.bind( wpcom ), site.ID, pluginData.id );
```

Comment:
what happens if we try to update a plugin that is already in its last version? Do the endpoint return an error?